osf-packager/src/main/java/org/dataconservancy/cos/packaging/OsfContentProvider.java

```diff
@@ -292,26 +173,45 @@ private static Node buildContentTree(Model domainObjects, RelationshipResolver r
     }
 
     /**
-     * Create a FileInfo that points to file content present at a URL.  The content from the URL is downloaded and
-     * stored in a temporary file.
+     * If they haven't already been initialized, generate the domain objects
+     * and store them in the member variable.
+     */
+    private void initDomainObjects() {
+        if (domainObjects != null)
+            return;
+
+        ByteArrayOutputStream sink = new ByteArrayOutputStream();
+        packageGraph.serialize(sink, RDFFormat.TURTLE_PRETTY, packageGraph.OSF_SELECTOR);
+
+        try {
+            LOG.debug("Packaging graph:\n{}", IOUtils.toString(sink.toByteArray(), "UTF-8"));
+        } catch (IOException e) {
+            // ignore
+        }
+
+        domainObjects = ModelFactory.createDefaultModel()
+                .read(new ByteArrayInputStream(sink.toByteArray()), null, "TTL");
+    }
+
+    /**
+     * Create a FileInfo that points to file content present at a URL.
+     * The content from the URL is downloaded and stored in a temporary file.
      * <p>
      * The logical name of the file represented in the FileInfo is the {@code name} parameter.
      * </p>
      *
      * @param filename the logical name of the content represented by the returned {@code FileInfo}
      * @param contentUrl resolvable URL to the content
-     * @param resolver used to resolve the {@code contentUrl} and download the content
```

Comment:
Sorry, the parameter is no longer needed because the resolver is a member.  I removed the doc line for this reason, but didn't get the parameter itself.